tests/Unit/Services/VoucherEvaluator/VoucherEvaluatorTest.php

```diff
@@ -176,6 +197,27 @@ public function itNoticesWhenAChildIsAlmostSchoolAge()
         $this->assertContains(self::NOTICE_TYPES['ChildIsAlmostSchoolAge'], $notices);
     }
 
+    /** @test */
+    public function itNoticesWhenAChildIsAlmostSecondarySchoolAge()
+    {
+        // Need to change the values we use for school start to next month's integer
+        Config::set('arc.school_month', Carbon::now()->addMonth(1)->month);
```

Comment:
If it is December and this month wraps around to January, this test will not behave as expected.

Reply:
As discussed, Card and put in tech debt to be dealt with next heartbeat.